lte/gateway/c/core/BUILD.bazel

```diff
@@ -1138,6 +1138,10 @@ cc_binary(
     srcs = [
         "oai/oai_mme/oai_mme.c",
     ],
+    copts = [
+        "-DEMBEDDED_SGW",  # TODO(#11638): Feature flags fixups
```

Comment:
Thanks for looking into this issue!! I think eventually (maybe soon), we should pull these copts to per_file_copts that can be controlled by a build time flag. Where the build time flag corresponds to the FEATURES we support in Makefile. see https://github.com/magma/magma/issues/11638

So something like `bazel build <MME TARGET> --config=of_mme`